datareduction/datareductionoperator.cpp

```diff
@@ -1436,6 +1436,136 @@ namespace DRO {
       return true;
    }
 
+   // Precipitation directional differential number flux
+   VariablePrecipitationDiffFlux::VariablePrecipitationDiffFlux(cuint _popID): DataReductionOperatorHasParameters(),popID(_popID) {
+      popName = getObjectWrapper().particleSpecies[popID].name;
+      cosAngle = cos(10.*M_PI/180.0); // cosine of fixed loss cone angle
```

Comment:
Should this not be a cfg parameter too? Or is 10° a Holy Value (TM) of the Ionospheric Community (TM)?

Reply:
I was wondering about that too. I can certainly make it a cfg parameter with 10 deg as a default value. I chose this value based on empirical considerations :)

Reply:
Then I strongly suggest to avoid hard-coded values and make it also a parameter for the user.

Reply:
And also then output as a .vlsv file parameter

Reply:
Good point!

Reply:
Thanks for bringing it up, I also add it as an output parameter in bulk files.